packages/orbit-components/src/Wizard/WizardStepIcon.js

```diff
@@ -0,0 +1,130 @@
+// @flow
+import * as React from "react";
+import styled, { css } from "styled-components";
+import convertHexToRgba from "@kiwicom/orbit-design-tokens/lib/convertHexToRgba";
+
+import Text from "../Text";
+import CheckCircle from "../icons/CheckCircle";
+import useMediaQuery from "../hooks/useMediaQuery";
+import useTheme from "../hooks/useTheme";
+import defaultTheme from "../defaultTheme";
+
+import type { Status } from ".";
+
+const centerMixin = css`
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+`;
+
+const StyledCircle = styled.div`
+  width: ${({ width }) => Math.round(width)}px;
+  height: ${({ height }) => Math.round(height)}px;
+  ${centerMixin};
+  background: ${({ theme }) => theme.orbit.paletteProductNormal};
+  border-radius: 50%;
+  ${({ status }) =>
+    status === "completed" &&
+    css`
+      background: ${({ theme }) => theme.orbit.paletteWhite};
+    `}
+  ${({ status }) =>
+    status === "disabled" &&
+    css`
+      background: ${({ theme }) => theme.orbit.paletteCloudNormalHover};
+    `}
+  ${({ active, hasGlow }) =>
+    active &&
+    hasGlow &&
+    css`
+      box-shadow: 0 0 0 4px ${({ theme }) => convertHexToRgba(theme.orbit.paletteProductNormal, 20)};
```

Comment:
You can destructure it everything on the top of the component :) it will be easier to read and add